dts/bindings/gpio/nxp,imx-gpio.yaml

```diff
@@ -29,6 +29,18 @@ properties:
         the order of the nodes matters. The first node for gpio1 will be used
         as the pinmux for gpio0, port 0.
 
+    gpio-gaps:
```

Comment:
Can't [`gpio-reserved-ranges`](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/gpio/gpio-controller.yaml#L29) be used for this?

Reply:
Yes it can- was looking for a property like that, don't know how I missed it. I'll update this PR.